float32 motorPosition
uint16 motorPositionSteps
